ddapm_test_agent/trace.py

```diff
@@ -275,11 +276,50 @@ def set_metric_tag(s: Span, k: str, v: MetricType) -> Span:
     return s
 
 
+def parse_json(json_string: bytes) -> Dict[str, Any]:
```

Comment:
can we add a docstring for why this method exists? I think it would help a future reader